.github/workflows/build.yml

```diff
@@ -8,11 +8,12 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
     strategy:
       matrix:
+        os: [ubuntu-latest, macos-latest]
```

Comment:
@vstinner 
We may able to add `windows-latest` but I didn't check pyperf test on the Windows machine.
Did you ever been to run tests on Windows?

Reply:
If there is a reliable Windows CI, why not. But for my own usage, Linux support is enough. I only try to provide best support for macOS and Windows.